en/includes/guides/service-extensions/pre-flow-extensions/pre-update-password-action.md

```diff
@@ -326,6 +326,36 @@ Content-Type: application/json
 }
 ```
 
+Below is an example of a failed response due to invalid password format validation.
+
+Response from external service:
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "actionStatus": "FAILED",
+  "failureReason": "invalid_value",
+  "failureDescription": "Invalid password format."
+}
```

Comment:
SInce this part is already added above, do we need to add it again ?

Reply:
This was added to show how the error responses coming from the external extension are mapped to the end response via recovery APIs. Do you think it is not required ?

Reply:
No I mean, same example for the Response from external service is already mentioned in (line 300-310).
I was thinking whether we could say the recovery API response by referencing to that as well without adding a new one

Reply:
yes we don't need to repeat. Just mention "When the password rest is initiated via the forgot password, forced password reset or user invitation to register setting password flow and gets updated with password reset API (better link the API), the failure results in the following error response being sent to the application."

Check for consistency of wording used already in the doc when referring to flows

Reply:
Fixed 4f22ed462493776baff60e33ea187361b6c40b58